contracts/Whitelist.sol

```diff
@@ -9,17 +9,22 @@ import "./packages/oz/Ownable.sol";
  * @notice The whitelist module keeps track of all valid Otoken contracts.
  */
 contract Whitelist is Ownable {
-    ///@notice mapping to track whitelisted product
+    /// @notice mapping to track whitelisted product
```

Comment:
Does your editor throw warnings here? We could change this tag from `@notice` to `@dev` 

```
Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
```